src/app/workflow/workflow.component.html

```diff
@@ -47,6 +47,26 @@ <h2 class="text-center">The workflow linked to could not be found!</h2>
       This {{ workflow.entryTypeMetadata.term }} was archived on {{ workflow.dbUpdateDate | date: 'yyyy-MM-dd' }}. It is now read-only.
     </mat-card>
   </div>
+  <div class="row m-1 mt-3 mb-0" *ngIf="!isWorkflowPublic && workflow.mode === 'DOCKSTORE_YML' && gitHubAppInstalled === false">
+    <mat-card class="alert alert-warning mat-elevation-z m-0">
```

Comment:
I believe we usually have a warning `mat-icon` for these warning banners like https://github.com/dockstore/dockstore-ui2/blob/30d4692c5b6ee264fe2e147da5517a953be8cdb9/src/app/workflow/workflow.component.html#L283